index.html

```diff
@@ -105,13 +105,27 @@ <h2>Implementations</h2>
 										<span class="icon solid fa-code-branch"></span>
                     <strong><a href="https://github.com/hyperledger/besu/tree/mega-eof">Besu</a></strong>
 									</li>
-                  <li class="style3">
-                    <span class="icon solid fa-code-branch"></span>
-                    <strong>EOF Tests</strong>
-                    <div><a href="https://github.com/ethereum/tests/tree/develop/src/EOFTestsFiller">- Validation Tests</a></div>
-                    <div><a href="https://github.com/ethereum/tests/tree/develop/src/EIPTestsFiller/StateTests/stEOF">- Execution Tests</a></div>
-                    </ul>
-                  </li>
+									<li class="style4">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/NethermindEth/nethermind/commits/feature/evm/eof">Nethermind 🚧</a></strong>
+									</li>
+									<li class="style5">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/ethereum/go-ethereum/pull/29518">Geth 🚧</a></strong>
+									</li>
+									<li class="style1">
+										<span class="icon solid fa-code-branch"></span>
+                    <strong><a href="https://github.com/ethereum/solidity/pull/13825">Solidity 🚧</a></strong>
+									</li>
+									</ul>
+									<ul class="statistics">
```

Comment:
Would link to ethereumjs too: https://github.com/ethereumjs/ethereumjs-monorepo/pull/2453

Reply:
oh yeah, it was all stale before, but after ACDE definitely need to include

Reply:
Alright can be done separately too